### PR TITLE
Navigation Component: Empty or undefined verticalKey has same behavio…

### PR DESCRIPTION
**Pull request by alexisgrow**
…r as null

If an implementer passed in a verticalKey that was empty or undefined, its
entry would be ignored in the navigation component. Changing the null check
so any falsy verticalKey is treated the same.

TEST=manual

Test on local answers instance that universal verticalPages config is
used when omitting vertical key.

### CODE REVIEW

File: src/ui/components/navigation/navigationcomponent.js [hunk 1]
```diff
@@ -99,7 +99,7 @@ export class Tab {
 
       // For tabs without config ids, map their URL to the configID
       // to avoid duplication of renders
-      if (tab.verticalKey === null && tabs[tab.verticalKey] === undefined) {
+      if (!tab.verticalKey && !tabs[tab.verticalKey]) {
```
Comment (oshi97) [inline src/ui/components/navigation/navigationcomponent.js]: I know it was like this in the old code, but do we need the `&& !tabs[tab.verticalKey]` ?

Comment (alexisgrow) [reply src/ui/components/navigation/navigationcomponent.js]: Oh good point, is this the logic we should have: 
`if (!tab.verticalKey && !tabs[tab.url])`?

Comment (oshi97) [reply src/ui/components/navigation/navigationcomponent.js]: yes that looks good to me!

**Review by oshi97 — APPROVED** [review]
lgtm!

**Pull request merged**